source/administration/statistics.rst

```diff
@@ -0,0 +1,70 @@
+Statistics
+================
+
+Statistics on users, posts and channels are tracked for each system and team. Enterprise Editions have access to advanced system statistics.
+
+To maximize performance for large Enterprise deployments, statistics for total posts, total hashtag posts, total file posts, posts per day, and active users with posts per day are disabled. You can re-enable them by changing the ``MaxUsersForStatistics`` value `in config.json <https://docs.mattermost.com/administration/config-settings.html#max-users-for-statistics>`_.
+
+For advanced metrics for Enterprise deployments, `see performance monitoring documentation to learn more <http://docs.mattermost.com/deployment/metrics.html>`_.
+
+Site statistics
+-----------------
+
+System statistics are viewable under **System Console** > **Reporting** > **Site Statistics**. The data shown here is a cumulative sum
+across all teams on the system.
+
+**Total Users:** The total number of active accounts created on your system. Excludes inactive accounts.
+
+**Total Teams:** The total number of teams created on your system.
+
+**Total Posts:** The total number of posts made in all the teams on your system, including deleted posts and posts made using automation.
+
+**Total Channels:** The total number of public channels and private groups created in all the teams on your system, including deleted channels. Doesn't include direct message channels.
+
+**Total Posts (graph):** The total number of posts made on a certain day in all the teams on your system, including deleted posts and posts made using automation.
+
+**Active Users With Posts (graph):** Users who made a post on a certain day in all the teams on your system, including system messages posted from the user's account.
+
```

Comment:
Would these be better as a definition list? The .rst markup for that looks like this:

    Total Users
      The total number of active accounts created on your system. Excludes inactive accounts.

    Total Teams
      The total number of teams created on your system.
    
    and so on

You wouldn't need the bold markers because the output to html is a \<dl\> element.